include/git2/worktree.h

```diff
@@ -180,7 +180,7 @@ typedef enum {
 typedef struct git_worktree_prune_options {
 	unsigned int version;
 
-	uint32_t flags;
+	git_worktree_prune_t flags;
```

Comment:
This is not correct, though. An enum should always have a valid enum value, only, and not be used as a field of bits. As we're ORing multiple flags into this field this has to bee an uint32.

Reply:
Right, I missed the ORing 🙄. Will drop from patchset. I was trying to get nice a link to show up, so I'll make it a doc-comment instead.